Automattic/wp-calypso
# Add selectors for adminMenu state

#### Changes proposed in this Pull Request

* Add selectors for retrieving `adminMenu` state.
* Add basic test coverage.

#### Testing instructions

* Checkout this PR.
* Run unit tests and check they pass `yarn run test-client:watch client/state/admin-menu/test`.

Fixes https://github.com/Automattic/wp-calypso/issues/45466
